package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "sjrk-story-telling",
-    "version": "0.4.0",
+    "version": "0.4.1",
```

Comment:
We didn't actually tag this one yet, so we don't actually need to do the version bump, but I'm not sure what the practice was before.

Reply:
There wasn't a practice before, in fact haha, but I bumped it to keep with semver. Should I drop it back down to 0.4.0?

Reply:
I suppose since we're pre-1.0.0 it doesn't affect things much, either way

Reply:
I've set it back to 0.4.0 :)